server/lib/pbench/indexer.py

```diff
@@ -3205,12 +3205,14 @@ def mk_sosreports(tb, extracted_root, logger):
             continue
         host_md = hostnames_if_ip_from_sosreport(os.path.join(extracted_root, sos))
         # get hostname (short and FQDN) from sosreport
-        d = _dict_const()
-        d['name'] = sos
-        d['md5'] = md5_val
-        d.update(host_md)
-        sosreportlist.append(d)
-
+        if 'OK' in host_md:
```

Comment:
`host_md` is now a tuple, so check with

       if host_md[0] == OK:

where OK is one value of the enumeration.